src/auditlog/models.py

```diff
@@ -94,6 +94,9 @@ def get_for_objects(self, queryset):
 
         if isinstance(primary_keys[0], integer_types):
             return self.filter(content_type=content_type).filter(Q(object_id__in=primary_keys)).distinct()
+        elif isinstance(queryset.model._meta.pk, models.UUIDField):
```

Comment:
If that is the case you can also do the following check:

```python
isinstance(primary_keys[0], uuid.UUID)
```

This is more in line with the check before by explicitly checking the type of the primary key value.

Another solution might be to always call smart_text on non-integer primary keys, but that might be unnecessary in most cases. Not sure myself which is better, I am sure you - as user of UUID fields - can choose something sensible here.